openfisca_israel/tests/pensions.yaml

```diff
@@ -0,0 +1,88 @@
+# Test files describe situations and their expected outcomes
+# We can run this test on our command line using `openfisca-run-test income_tax.yaml`
+
+- name: Pension for women
+  period: 2018-01
+  input_variables:
+    gender: woman
+    pension_contributing_years: 22
+    is_resident: true
+    born_in_israel: true
+  output_variables:
+    eligible_for_pension: true
+    pension_eligibility_age: 62
+
+- name: Pension for men
+  period: 2018-01
+  input_variables:
+    gender: man
+    pension_contributing_years: 22
+    is_resident: true
+    born_in_israel: true
+  output_variables:
+    eligible_for_pension: true
+    pension_eligibility_age: 67
+
+- name: Must be a resident
+  period: 2018-01
+  input_variables:
+    gender: man
+    pension_contributing_years: 22
+    is_resident: false
+    born_in_israel: true
+  output_variables:
+    eligible_for_pension: false
+
+- name: Must contribute for 12 years
+  period: 2018-01
+  input_variables:
+    gender: man
+    pension_contributing_years: 11
+    is_resident: true
+    born_in_israel: true
+  output_variables:
+    eligible_for_pension: false
+
+- name: Woman under 62 when immigrated
+  period: 2018-01
+  input_variables:
+    gender: woman
+    pension_contributing_years: 22
+    is_resident: true
+    born_in_israel: false
+    age_at_immigration: 50
+  output_variables:
+    eligible_for_pension: true
+
+- name: Woman over 62 when immigrated
+  period: 2018-01
+  input_variables:
+    gender: woman
+    pension_contributing_years: 22
+    is_resident: true
+    born_in_israel: false
+    age_at_immigration: 61
+  output_variables:
+    eligible_for_pension: true
+
+- name: Man under 62 when immigrated
+  period: 2018-01
+  input_variables:
+    gender: man
+    pension_contributing_years: 22
+    is_resident: true
+    born_in_israel: false
+    age_at_immigration: 50
+  output_variables:
+    eligible_for_pension: true
+
+- name: Man over 62 when immigrated
+  period: 2018-01
+  input_variables:
+    gender: man
+    pension_contributing_years: 22
+    is_resident: true
+    born_in_israel: false
+    age_at_immigration: 63
+  output_variables:
+    eligible_for_pension: true
```

Comment:
Ooops. This should be false

Reply:
@Br3nda why should this be false? is it because his `age_at_immigration`should be less than 63?